lim_test.py

```diff
@@ -0,0 +1,259 @@
+'''tests for the longintmath.py module'''
+import unittest
+import longintmath as lim
+
+class TestLIMFunc(unittest.TestCase):
```

Comment:
i'm not sure about the actual function of the classes in a test suite.  i just took a stab.  is it usually one class per module?